components/button/button.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>button</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="button.css">
+    <link rel="stylesheet" href="../common.css">
+     <!-- font-awesome-cdn -->
+     <script src="https://kit.fontawesome.com/c636cd5029.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <h1>Button</h1>
+    <p>By using below mentioned css classes, you can mirror the below components. </p>
+    <h3>Primary, secondary and link buttons:</h3>
+    <div class="flex flex-gap-2">
+        <button class="button btn-primary btn-cursor">Primary</button>
+        <button class="button btn-danger btn-cursor">Delete</button>
+        <button class="button btn-secondary btn-cursor">Secondary</button>
```

Comment:
Insted of creating different class like "btn-cursor" you can define pointer to button itself

Reply:
ok